### PR TITLE
Monorepo: update READMEs, other content

### PR DESCRIPTION
**Pull request by silochad**

### CODE REVIEW

**Comment by silochad** [comment]
Capitalizing text filenames so GitHub shows them at the top of the file list for discoverability

File: .github/ISSUE_TEMPLATE/ui_bug_report.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
-name: Bug report
-about: File a bug report
+name: 'UI: Bug report'
```
Comment (hellofromguy) [inline .github/ISSUE_TEMPLATE/ui_bug_report.md:2]: do these templates work if the .github/ folder is not in the root directory?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+[discord-url]: https://discord.gg/beanstalk
+
+[proj-protocol]: https://github.com/BeanstalkFarms/Beanstalk/tree/master/protocol
+[proj-sdk]: https://github.com/BeanstalkFarms/Beanstalk/tree/master/projects/sdk
+[proj-ui]: https://github.com/BeanstalkFarms/Beanstalk/tree/master/projects/ui
+[proj-subgraph-beanstalk]: https://github.com/BeanstalkFarms/Beanstalk/tree/master/projects/subgraph-beanstalk
+[proj-subgraph-bean]: https://github.com/BeanstalkFarms/Beanstalk/tree/master/projects/subgraph-bean
+[proj-cli]: https://github.com/BeanstalkFarms/Beanstalk/tree/master/projects/cli
+[es-beanstalk]: https://etherscan.io/address/0xC1E088fC1323b20BCBee9bd1B9fC9546db5624C5
+[es-bean]: https://etherscan.io/address/0xBEA0000029AD1c77D3d5D23Ba2D8893dB9d1Efab
```
Comment (hellofromguy) [inline README.md:11]: should these links be relative paths so they work on non-master branches?

File: .github/ISSUE_TEMPLATE/ui_bug_report.md (hunk 1 above)
Comment (silochad) [reply .github/ISSUE_TEMPLATE/ui_bug_report.md:2]: Unfortunately not AFAIK, they weren't working after merge, so moved the UI Bug report template down to the root dir and got rid of the UI feature request for now until we come up with a better structure around how feature requests should be presented

**Pull request merged**